wrappers/python/nntile/model/gpt2.py

```diff
@@ -320,7 +321,11 @@ def __init__(
         self.num_hidden_layers = num_hidden_layers
         self.next_tag = next_tag
         # Fill Base Model with the generated data
+        print('before super: ', self.config)
         super().__init__(activations, layers)
+        print('after super: ', self.config)
```

Comment:
It is better to use gpt2lmhead model, as gpt2 is deprecated. GPT2 is still here, as not all functionality as moved to GPT2LMHeadModel